build.gradle

```diff
@@ -55,12 +55,14 @@ buildscript {
 }
 
 allprojects {
-  group = 'com.squareup.okio'
-  version = '1.15.0-SNAPSHOT'
+  group = GROUP
+  version = VERSION_NAME
 }
 
 subprojects {
   repositories {
     mavenCentral()
   }
+
+  apply from: "$rootDir/gradle/gradle-mvn-push.gradle"
```

Comment:
I'm not sure we should do this as it causes the bechmarks, samples, and root project to be pushed. Maven sort of forced that on us, but there's no reason to push those things with Gradle.